boolebots/src/components/Speed.js

```diff
@@ -0,0 +1,22 @@
+import React, {useState, useEffect} from 'react'
+
+export default function Speed() {
+    
+const [speed, setSpeed] = useState(1)
+
+ useEffect(() => {
+    const interval = setInterval(() => {
+      // Move bots here with the current speed value
+    }, 1000 / speed); // Adjust bot movement speed based on the selected speed
+
+    return () => clearInterval(interval);
+  }, [speed]);
+
+    return (<input 
```

Comment:
I think maybe some kind of indication of speed would be helpful? Like have below it say slow med fast.

Reply:
Even just something like this (but it should probably say what they'r
![Screen Shot 2023-05-15 at 2 59 23 PM](https://github.com/chingu-voyages/v44-tier2-team-25/assets/102242515/3a8d3c80-3f19-4845-a26f-378c821baad3)
e in seconds)

Reply:
Hey Matt. Thanks for the review. I have added this speed indicator and also I forgot to render the operation drop-down.